src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a href="#home" class="logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+        <nav class="nav">
+          <ul>
+            <li class="nav-item">
+              <a href="#" class="nav-link underline is-active">Apple</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-link">Samsung</a>
+            </li>
```

Comment:
```suggestion
            <li class="nav-item">
              <a href="#" class="nav-link underline is-active">Apple</a>
            </li>

            <li class="nav-item">
              <a href="#" class="nav-link">Samsung</a>
            </li>
```

Add blank line between multiple sibling blocks